src/test/java/org/jboss/xavier/analytics/rules/BaseIntegrationTest.java

```diff
@@ -43,4 +47,13 @@ public void setup()
     public void tearDown()
     {
     }
+
+    public <T> List<T> extractModels(Map<String, Object> results, Class<T> model)
```

Comment:
This code is duplicated in BaseTest. It could be refactored and extract a common super class in order to reduce duplication ?

Reply:
This rang a bell in my head since i was sure about having already implemented this method and, in fact, it was in the `Utils` class so i refactored for having that to be used